source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -267,6 +275,15 @@ void AuthenticatorImpl::doneWithStatus(const Status& status) {
   ENVOY_LOG(debug, "{}: JWT token verification completed with: {}", name(),
             ::google::jwt_verify::getStatusString(status));
 
+  if (status == Status::Ok) {
+    auto token_exp = absl::ToUnixSeconds(absl::Now());
+    if (jwks_data_->getJwtProvider().has_token_cache_duration()) {
+      token_exp +=
+          DurationUtil::durationToSeconds(jwks_data_->getJwtProvider().token_cache_duration());
+    }
```

Comment:
we need to check jwt_.exp_ field.  if min(jwt.exp_,  cache_duration).

Or we don't need `cache_duration` config, always use jwt.exp field

Reply:
Removed cache_duration.